pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -51,14 +52,21 @@ type queueAttr struct {
 	allocated *api.Resource
 	request   *api.Resource
 	// inqueue represents the resource request of the inqueue job
-	inqueue    *api.Resource
-	capability *api.Resource
+	inqueue        *api.Resource
+	capability     *api.Resource
+	realCapability *api.Resource
```

Comment:
Suggest to add comment to describe the difference between `capability` and `realCapability`?